docs/source/estimator_intro.ipynb

```diff
@@ -128,7 +128,7 @@
     "UMAP is a dimensionality reduction algorithm which performs non-linear dimension reduction. It can also be used for visualization.\n",
     "For additional information on the UMAP model please refer to the documentation on https://docs.rapids.ai/api/cuml/stable/api.html#cuml.UMAP\n",
     "\n",
-    "Trustworthiness is a measure of the extent to which the local structure is retained in the embedding of the model. Therefore, if a sample predicted by the model lied within the unexpected region of the nearest neighbors, then those samples would be penalized. For more information on the trustworthiness metric please refer to: https://scikit-learn.org/dev/modules/generated/sklearn.manifold.t_sne.trustworthiness.html\n",
+    "Trustworthiness is a measure of the extent to which the local structure is retained in the embedding of the model. Therefore, if a sample predicted by the model lay within the unexpected region of the nearest neighbors, then those samples would be penalized. For more information on the trustworthiness metric please refer to: https://scikit-learn.org/dev/modules/generated/sklearn.manifold.t_sne.trustworthiness.html\n",
```

Comment:
```suggestion
    "Trustworthiness is a measure of the extent to which the local structure is retained in the embedding of the model. Therefore, if a sample predicted by the model lays within the unexpected region of the nearest neighbors, then those samples would be penalized. For more information on the trustworthiness metric please refer to: https://scikit-learn.org/dev/modules/generated/sklearn.manifold.t_sne.trustworthiness.html\n",
```

Reply:
Haha, feedback is much appreciated, whenever it comes!

I don't think this suggestion is correct, however. "lay" is the subjunctive past conjugation of lie. In this case, we could correctly use "lay" if we want to keep it past tense as before, or "lies" if we want to move it to present tense and remove the subjunctive mood. Or we could do "lie" if we want to do present tense and keep the subjunctive mood, but that's a fairly unusual construction in contemporary American English.

"lays" would be a conjugation of "to lay" as opposed to "to lie" and is a transitive verb with a meaning we don't intend here. e.g. "He lays his head down to sleep"

And hooray! My English degree has finally been useful in a PR!

Reply:
Fair enough, technically speaking English is my second language (even if I've spoken it since forever, pretty much) :)